airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -397,10 +397,10 @@ def deferrable_create_cluster_next(self, context: Context, event: dict[str, Any]
                 )
 
     def execute_failed(self, context: Context, event: dict[str, Any] | None = None) -> None:
-        if event is None:
-            self.log.info("Trigger error: event is None")
-            raise AirflowException("Trigger error: event is None")
-        elif event["status"] == "delteted":
+        if event is None or event["status"] != "deleted":
+            self.log.info("Trigger error: event is %s", event)
+            raise AirflowException(f"Trigger error: event is {event}")
+        else:
             self.log.info("Cluster deleted")
             raise event["exception"]
```

Comment:
I've just wondering what is the source of "exception", because it should return this object ideally
https://github.com/apache/airflow/blob/609eed90698aa7abeb5bae9ae156062d32baae86/airflow/providers/amazon/aws/triggers/eks.py#L139
And for me it `else` branch is unreachable right now, but maybe I miss something

Reply:
I'm working on writing some unit tests to test the working of the method (new to Airflow, so it's a learning curve)